source/production/F0.Minesweeper.Components/Cell.razor.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Threading.Tasks;
+using F0.Minesweeper.Logic.Abstractions;
+using Microsoft.AspNetCore.Components;
+
+namespace F0.Minesweeper.Components
+{
+	public partial class Cell
+	{
+		[Parameter]
+		public Location Location { get; set; }
+		private string StatusText
+		{
+			get => statusText;
+			set
+			{
+				if (statusText == value)
```

Comment:
We could shorten the `set` block, if we would invert the `if` statement.

Reply:
So so true :) Usually I like the current "gated" style better but for such properties doing it your way is definitely nicer